source/services/dataContracts/converters/aliasConverter/aliasConveter.tests.ts

```diff
@@ -43,4 +44,33 @@ describe('aliasConverter', (): void => {
 		sinon.assert.calledOnce(testConverter.toServer);
 		sinon.assert.calledWith(testConverter.toServer, 'value');
 	});
+
+	describe('integrationTest', (): void => {
+		it('should apply an alias as part of a transform mapping', (): void => {
+			let testConverter = {
+				fromServer(value: number): number { return value + 5; },
+				toServer(value: number): number { return value - 5; },
+			};
+
+			let transform: any = {
+				value: new AliasConverter('valueFromServer'),
+				number: new AliasConverter('numberFromServer', testConverter),
+			};
```

Comment:
So per this, basically the purpose of the alias converter is to map server values that have different property names than the corresponding client properties?